runtime/vara/src/weights/frame_system.rs

```diff
@@ -53,27 +53,27 @@ impl<T: frame_system::Config> frame_system::WeightInfo for SubstrateWeight<T> {
         // Proof Size summary in bytes:
         //  Measured:  `0`
         //  Estimated: `0`
-        // Minimum execution time: 1_503_000 picoseconds.
-        Weight::from_parts(1_491_824, 0)
+        // Minimum execution time: 1_619_000 picoseconds.
+        Weight::from_parts(1_222_978, 0)
             // Standard Error: 0
-            .saturating_add(Weight::from_parts(459, 0).saturating_mul(b.into()))
+            .saturating_add(Weight::from_parts(460, 0).saturating_mul(b.into()))
     }
```

Comment:
**Suggestion:** The execution time for the functions `remark`, `remark_with_event`, `set_heap_pages`, `kill_storage`, and `kill_prefix` have been increased. It's recommended to investigate the reason for this increase and try to optimize the functions if possible.
```suggestion
        The code optimization should be done based on the specific logic of these functions. It's not possible to provide a generic optimized code snippet.
```